lib/internal/test_runner/harness.js

```diff
@@ -152,7 +152,7 @@ function setup(root) {
 
   const terminationHandler = async () => {
     await exitHandler();
-    process.exit();
+    process.exit(1);
```

Comment:
changed to `1` as otherwise this test failed:
https://github.com/nodejs/node/blob/384e1b5174976be75432184e0530b7cb5b00b124/test/parallel/test-runner-exit-code.js#L25

the exit code is equal to 0 instead of 1


weird it worked before...

Reply:
I feel strongly like this should not be changed

Reply:
I agree, changed it here until I fix the rest of the things